mlir/include/mlir/ExecutionEngine/SparseTensor/Storage.h

```diff
@@ -349,6 +349,8 @@ class SparseTensorStorage final : public SparseTensorStorageBase {
 
   ~SparseTensorStorage() final = default;
 
+  void sortInPlace();
```

Comment:
can  you
(1) move this towards the end of all methods in public part (since we should have constructor/destructor/getters/setters and then other methods grouped by logic as order)
(2) add a comment